jenkins/node-image/node-runner.sh

```diff
@@ -27,5 +27,11 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
 fi
 
+echo "ls .."
+ls ..
+
+echo "ls"
+ls
+
 # Run script
-"${NODE_SCRIPT:-test/e2e_node/jenkins/e2e-node-jenkins.sh} ${NODE_PROPERTY:-test/e2e_node/jenkins/jenkins-ci.properties}"
+"${NODE_TEST_SCRIPT} ${NODE_TEST_PROPERTIES}"
```

Comment:
this should probably be 
```bash
"${NODE_TEST_SCRIPT}" "${NODE_TEST_PROPERTIES}"
```

With the single set of quotes, bash is trying to treat the whole thing as one command (with a space in the name)